lib/fluent/plugin/out_google_cloud.rb

```diff
@@ -967,12 +968,16 @@ def set_http_request(record, entry)
 
       latency = input.delete('latency')
       unless latency.nil?
-        # Parse latency to duration. If failed, skip setting latency.
-        duration_in_seconds = ChronicDuration.parse(latency)
-        if duration_in_seconds
+        # Parse latency. If no valid format is detected, skip setting latency.
+        # Format: space/tab (optional) + integer + point & decimal (optional)
+        #       + space/tab (optional) + "s" + space/tab (optional)
+        # e.g.: "1.42 s"
+        parsed_groups = latency.scan(LATENCY_REGEX)
+        if !parsed_groups.empty? && parsed_groups[0].length == 2
```

Comment:
You have exactly two captures in the string. If there's a match (`parsed_groups` is non-empty), when will `parsed_groups[0].length` be anything but 2?